packages/web/src/generic.test.ts

```diff
@@ -39,3 +39,83 @@ describe('inject', () => {
     });
   });
 });
+
+describe('track custom events', () => {
```

Comment:
Just noticed that we're missing two describe blocks for `'in development mode'` and `'in production mode'` to test the separate modes